charts/gateway/values.yaml

```diff
@@ -370,8 +370,8 @@ management:
     type: LoadBalancer
     ## Load Balancer sources
     ## https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/#restrict-access-for-loadbalancer-service
-    annotations: {}
-      # cloud.google.com/load-balancer-type: "Internal"
+    annotations:
+      cloud.google.com/load-balancer-type: "Internal"
```

Comment:
I forgot to comment this back

Reply:
Resolved by https://github.com/CAAPIM/apim-charts/pull/193/commits/32daaae94b8d7f0ffc7f9d15b56c23e82e7e7ace